src/module/apps/compendium-browser/settings.ts

```diff
@@ -0,0 +1,199 @@
+import { htmlQuery, htmlQueryAll, localizer, objectHasKey } from "@util";
+import type {
+    HandlebarsRenderOptions,
+    HandlebarsTemplatePart,
+} from "types/foundry/client-esm/applications/api/handlebars-application.ts";
+import type { PackInfo, TabName } from "./data.ts";
+
+const foundryApp = foundry.applications.api;
+
+class CompendiumBrowserSettingsApp extends foundryApp.HandlebarsApplicationMixin(foundryApp.ApplicationV2) {
```

Comment:
A separate app?

Reply:
The settings tab is taking up real estate and doesn't really fit in the same list as the content tabs. Also I don't think it's used nearly as much as the other tabs so it makes sense to move it out of the way.